src/components/Detailpage/Detailpage.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Detailpage from './Detailpage'
+
+export default {
+  title: 'Detail-Page',
+  component: Detailpage,
+}
+
+const title = 'Like meat - Like Nuggets'
+const image =
+  'https://raw.githubusercontent.com/denniswitzel/capstone-project/master/src/images/nuggets/Like-meat-Nuggets.png'
```

Comment:
Consider saving a local copy of this image to avoid this external request and external dependency